packages/core/src/styles/fluo.scss

```diff
@@ -1,52 +1,160 @@
+
+  --fl-border-radius-small: 0.2rem;
+  --fl-border-radius-medium: 0.4rem;
+  --fl-border-radius-large: 0.8rem;
+
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  // Typography tokens
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+  // Fonts
+  --fl-font-sans: "Roobert", sans-serif;
+  // Font sizes
+  --fl-font-size-x-small: 1rem;
+  --fl-font-size-small: 1.2rem;
+  --fl-font-size-medium: 1.6rem;
+  --fl-font-size-large: 2.4rem;
+  --fl-font-size-x-large: 3.8rem;
+  --fl-font-size-xx-large: 5.2rem;
+  --fl-font-size-xxx-large: 7.2rem;
+  // Font weights
+  --fl-font-weight-light: 300;
+  --fl-font-weight-regular: 400;
+  --fl-font-weight-medium: 500;
+  --fl-font-weight-semibold: 600;
+  --fl-font-weight-bold: 700;
+
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  // Transition tokens
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+  --fl-transition-x-slow: 1000ms;
+  --fl-transition-slow: 500ms;
+  --fl-transition-medium: 250ms;
+  --fl-transition-fast: 150ms;
+  --fl-transition-x-fast: 50ms;
+
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  // Input tokens
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+  // Height
+  --fl-input-height-small: 2.5rem;
+  --fl-input-height-medium: 3.5rem;
+  --fl-input-height-large: 5rem;
+  // Border width
+  --fl-input-border-width: 0.2rem;
+
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  // Default tag definitions
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+  font-size: 62.5%;
+  font-family: var(--fl-font-sans);
+  line-height: 1.5;
+
+  h1 {
+    font-size: var(--fl-font-size-xxx-large);
+    font-weight: var(--fl-font-weight-semibold);
+  }
+
+  h2 {
+    font-size: var(--fl-font-size-xx-large);
+    font-weight: var(--fl-font-weight-medium);
+  }
+
+  h3 {
+    font-size: var(--fl-font-size-x-large);
+    font-weight: var(--fl-font-weight-medium);
+  }
+
+  h4 {
+    font-size: var(--fl-font-size-large);
+    font-weight: var(--fl-font-weight-medium);
+  }
+
+  h5 {
+    font-size: var(--fl-font-size-medium);
+    font-weight: var(--fl-font-weight-medium);
+  }
+
+  h6 {
+    font-size: var(--fl-font-size-small);
+    font-weight: var(--fl-font-weight-medium);
+    text-transform: uppercase;
+  }
+
+  p {
+    font-size: var(--fl-font-size-medium);
+    font-weight: var(--fl-font-weight-regular);
+  }
```

Comment:
I'm not too convinced with these default values. For example, on the demo-1 branch, I had to uncomment these defaults because they were oversized and out of proportion.

No need to fix this for now, we can save this discussion for #9 

I really like the naming scheme! Easy to understand and remember

Reply:
resolved through our discussion yesterday. Can you comment out the assignments for h1-h6, p for the time-being?